packages/@okta/vuepress-site/docs/reference/api/roles/index.md

```diff
@@ -10,7 +10,7 @@ meta:
 
 The Okta Administrator Roles API provides operations to manage administrative Role assignments for a User.
 
-Role listing APIs provide a union of both standard and Custom Roles assigned to a User or Group. We are, therefore, providing both groups of APIs and their intersections in this single doc.
+Role listing APIs provide a union of both standard and Custom Roles assigned to a User or Group. Okta provides both groups of APIs and their intersections in this doc.
```

Comment:
What are the two APIs? We should name them explicitly here. @vasishtajayanti-okta

Reply:
@brentschaus-okta this is talking about the two _groups_ of APIs one for standard roles, which generally fall under, `api/v1/users|groups/{id}/roles*` and one for Custom Roles which fall under `/api/v1/iam/*`.
The document lists out all the apis below, so I think we don't need that here.